doc/ug/design.md

```diff
@@ -70,37 +70,52 @@ Capturing fast and efficient feedback on syntactic and semantic (as well as styl
 
-For other developers that do not have access to this tool, the recommendation is to run their code through whatever linting tool they have available at their disposal before creating a design Pull Request (e.g. the open source Verilator tool).
-Once the Pull Request has been merged, these developers can then work with the maintainers to enable sign-off linting on their design.
+Note that a pull request can not be merged if it is not lint-clean, since the continuous integration infrastructure will run Verible lint on each pull request.
 
-Note that all designs with enabled AscentLint targets will be run through the tool in eight-hour intervals and the results are published as part of the tool dashboards on the [hardware IP overview page](https://docs.opentitan.org/hw), enabling designers to close the lint errors and warnings even if they cannot run the sign-off tool locally.
+### Semantic Linting using AscentLint (Sign-Off)
 
-Goals for linting closure per design milestone are given in the [OpenTitan Development Stages]({{< relref "doc/project/development_stages" >}}) document.
+The rule set and capabilities of commercial tools currently still go beyond what open-source tools can provide.
+Hence, we have standardized on the [AscentLint](https://www.realintent.com/rtl-linting-ascent-lint/) tool from RealIntent for sign-off.
+This tool exhibits fast run-times and a comprehensive set of rules that provide concise error and warning messages.
 
-### Style Linting using Verible
+The sign-off lint flow leverages a new lint rule policy named _"lowRISC Lint Rules"_ that has been tailored towards our [Verilog Style Guide](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md).
+The lint flow run scripts and waiver files are available in the GitHub repository of this project but, due to the proprietary nature of the lint rules and their configuration, the _"lowRISC Lint Rules"_ lint policy file can not be publicly provided.
+However, the _"lowRISC Lint Rules"_ are available as part of the default policies in AscentLint release 2019.A.p3 or newer (as `LRLR-v1.0.policy`).
+This allows designers with access to this tool to run the lint flow locally on their premises.
 
-In order to complement the sign-off lint flow explained above, we also leverage the Verible style linter, which captures different aspects of the code and detects style elements that are in violation with our [Verilog Style Guide](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md).
+If developers of design IP have access to AscentLint, we recommend to set up the AscentLint flow for their IP as described in the [Lint Flow README]({{< relref "hw/lint/doc/README.md" >}}), and use the flow locally to close the errors and warnings.
+Linting errors and warnings can be closed by fixing the code in question (preferred), or waiving the error.
+These waivers have to be reviewed as part of the Pull Request review process.
```

Comment:
Nit: I think you've un-capitalized "Pull Request" elsewhere.

Reply:
I did a quick search and found another instance of this. I have un-capitalized all instances now.